addons/calendar/static/src/js/calendar_renderer.js

```diff
@@ -131,6 +148,17 @@ const AttendeeCalendarRenderer = CalendarRenderer.extend({
         }
         return selector;
     },
+    _installAndConfig: function (externalCalendarProvider) {
```

Comment:
"configureExternalCalendar" ?

The fact that it "installs" is more of an implementation details of our sub-modules structure.